tests/byzantium/eip196_ec_add_mul/test_ecadd.py

```diff
@@ -27,8 +27,106 @@
             Spec.G1,
             id="generator_plus_inf",
         ),
+        pytest.param(
+            Spec.INF_G1 + Spec.G1,
+            Spec.G1,
+            id="inf_plus_generator",
+        ),
+        pytest.param(
+            Spec.G1 + Spec.INF_G1,
+            Spec.G1,
+            id="generator_plus_inf_extra_inf",
+        ),
+        pytest.param(
+            Spec.INF_G1 + Spec.G1 + Spec.INF_G1,
+            Spec.G1,
+            id="inf_plus_generator_extra_inf",
+        ),
+        pytest.param(
+            b"",
+            Spec.INF_G1,
+            id="empty",
+        ),
+        pytest.param(
+            Spec.INF_G1,
+            Spec.INF_G1,
+            id="single_inf",
+        ),
+        pytest.param(
+            Spec.INF_G1 + Spec.INF_G1,
+            Spec.INF_G1,
+            id="double_inf",
+        ),
+        pytest.param(
+            Spec.INF_G1 + Spec.INF_G1 + Spec.INF_G1,
+            Spec.INF_G1,
+            id="triple_inf",
+        ),
+        pytest.param(
+            b"\0" * 80,
+            Spec.INF_G1,
+            id="80_zero_bytes",
+        ),
+        pytest.param(
+            Spec.G1,
+            Spec.G1,
+            id="single_generator",
+        ),
+        pytest.param(
+            Spec.G1 + Spec.G1,
+            Spec.G1x2,
+            id="double_generator",
+        ),
+        pytest.param(
+            Spec.G1 + Spec.G1 + Spec.INF_G1,
+            Spec.G1x2,
+            id="double_generator_extra_inf",
+        ),
```

Comment:
```suggestion
        ),
        pytest.param(
            Spec.G1 + Spec.G1 + PointG1(1, 3),
            Spec.G1x2,  # Extra invalid point is ignored
            id="double_generator_extra_pt_1_3",
        ),
```
Another test to make sure that an extra invalid point is successfully ignored.